File: VB.NET/Aula02_atividade02/Aula02_atividade02/Aluno.vb
Public Class Aluno
    Inherits Pessoa
    Private rgm As String

    Public Sub New(nome As String, rgm As String)
        MyBase.New(nome)
        Me.rgm = rgm
    End Sub

    Public Property prgm As String
        Get
            Return rgm
        End Get
        Set(value As String)
            rgm = value
        End Set
    End Property
End Class

File: C#/Aula01/Aula01/Aluno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Aula01
{
    class Aluno:Pessoa
    {
        private String rgm;

        public String RGM
        {
            get { return rgm; }
            set { rgm = value; }
        }

        public Aluno(String cpf, String nome, String rgm)
        {
            this.CPF = cpf;
            this.Nome = nome;
            this.RGM = rgm;
        }
    }
}
